build/build.c

```diff
@@ -16,6 +16,8 @@
 
 #include "debug.h"
 
+#define RPMRC_MISSINGBUILDREQUIRES 11
```

Comment:
Hmm, we're actually returning this in the API so I think it should actually go into rpmbuild.h

Reply:
Oh, this acentially moved to the _internal header. Will move again...